syntax/dashboard.sublime-syntax

```diff
@@ -0,0 +1,87 @@
+%YAML 1.2
+---
+# http://www.sublimetext.com/docs/3/syntax.html
+# This is a base syntax which is used in
+# `status`, `branch`, `rebase` and `tags`
+name: GitSavvy Dashboard
+hidden: true
+scope: git-savvy.dashboard
+contexts:
+  main:
+    - match: "^  BRANCH:"
+      comment: Summary Header
+      scope: comment.git-savvy.status.summary-header.title.branch
+      push:
+        - meta_scope: meta.git-savvy.status-summary
+        - match: ^$
+          pop: true
+
+        - match: ^  (ROOT:)(.+)
+          comment: root summary
+          scope: comment.git-savvy.status.root-summary
+          captures:
+            1: comment.git-savvy.status.summary-header.title.root
+            2: comment.git-savvy.status.root-summary.root.info
+
+        - match: "(`)([^`]+)(`)"
+          comment: branch names
+          scope: constant.other.git-savvy.status.branch
+          captures:
+            1: punctuation.definition.git-savvy.branch.start
+            2: constant.other.git-savvy.status.branch-name
+            3: punctuation.definition.git-savvy.branch.end
+
+        - match: (ahead)|(behind)
+          comment: ahead or behind
+          scope: keyword.other.git-savvy.status.ahead-behind
+
+        - match: ^  (HEAD:)\s+([0-9a-f]{7,40}) .+
+          comment: head summary
+          scope: comment.git-savvy.status.head-summary
+          captures:
+            1: comment.git-savvy.status.summary-header.title.head
+            2: constant.other.git-savvy.status.sha1
+
+    - match: "^  (#{3,})"
+      comment: Key-bindings menu
+      captures:
+        1: punctuation.definition.git-savvy.section
+      push:
+        - meta_scope: comment.git-savvy.status.key-bindings-menu
+
+        - match: ^-\n$
+          scope: comment.git-savvy.key-bindings-menu.end
+          pop: true
+
+        - match: "#{3,}"
```

Comment:
This is the part where we mark the help menu section and the `#`s around the help headers. it's required in almost all the dashboards.